inspectit-oce-bootstrap/src/main/java/rocks/inspectit/oce/bootstrap/context/NoopContextManager.java

```diff
@@ -0,0 +1,25 @@
+package rocks.inspectit.oce.bootstrap.context;

```

Comment:
`.noop` package, then we could always have all Noop stuff in the `.noop` packages?